src/tokens/builder.rs

```diff
@@ -133,6 +133,31 @@ impl<'a> PasetoBuilder<'a> {
     self
   }
 
+  /// Sets multiple arbitrary claims by passing a HashMap of keys and values (keys inside the json token).
+  /// If a key in our passed map already exists in the internal claims, the value will be updated
+  /// with the value of the key in the HashMap we pass in this method.
+  /// For more info, reference the std::iter::Extend trait documentaton at
+  /// https://doc.rust-lang.org/std/iter/trait.Extend.html
+  pub fn set_claims(&'a mut self, claims: HashMap<&'a str, Value>) -> &'a mut Self {
```

Comment:
Can we change this to be `extend_claims` to be more inline with the standard methods?:

```suggestion
  pub fn extend_claims(&'a mut self, claims: HashMap<&'a str, Value>) -> &'a mut Self {
```

Reply:
Done.  Great suggestion.  I love that.